src/Forms/TreeDropdownField.php

```diff
@@ -622,26 +621,22 @@ public function getAttributes()
     /**
      * HTML-encoded label for this node, including css classes and other markup.
      *
-     * @deprecated 4.0.1 Use setTitleField() instead
```

Comment:
Undeprecating this because setTitleField uses $this->titleField instead of $this->labelField